src/Core/src/Core.csproj

```diff
@@ -37,6 +37,7 @@
     <PackageReference Include="Xamarin.AndroidX.Navigation.Fragment" />
     <PackageReference Include="Xamarin.AndroidX.Navigation.Runtime" />
     <PackageReference Include="Xamarin.AndroidX.Navigation.Common" />
+    <PackageReference Include="Xamarin.AndroidX.Lifecycle.Process" Version="2.5.1.2"/>
```

Comment:
@mattleibow  Hi

I've redone it using `Xamarin.AndroidX.Lifecycle.Process`.  how should I install the package version correctly?
Is it also possible that you will do the review again? 

Thanks

Reply:
You can add the package reference update + version here: https://github.com/dotnet/maui/blob/main/eng/AndroidX.targets